### ISSUE TITLE
Static routing broken after 2022.4.2 update

### ISSUE THREAD

**Issue by VikeDragon** [home-assistant/supervisor#3565]
### Describe the issue you are experiencing

The wlan0 connection has been renamed as "Supervisor wlan0" and it is impossible to specify routing, since the connection is re-created when rebooting. So I can't lower the metric for wlan0 from 600 to 203, and previously I did routing and everything worked.
The Supervisor also noticed the update to version 2022.4.0, which took place in parallel and saw entries in the logs

WARNING (Main Thread) [supervisor.jobs ] 'The update '... different modules ...' blocked from execution, no supervisor internet connection

### What is the used version of the Supervisor?

2022.4.0

### What type of installation are you running?

Home Assistant Supervised

### Which operating system are you running on?

Debian

### What is the version of your installed operating system?

5.15

### What version of Home Assistant Core is installed?

2022.4.2

### Steps to reproduce the issue

update to version 2022.4.2 and superviser update to 2022.4.0

### Anything in the Supervisor logs that might be useful for us?

```txt
WARNING (Main Thread) [supervisor.jobs ] 'The update '... different modules ...' blocked from execution, no supervisor internet connection
```

**Comment by Mick51** [comment]
![Screenshot_20220412-213856_Home Assistant](https://user-images.githubusercontent.com/40418405/163040631-38f0dc09-2cb2-42a7-be3c-68fb9a8d372e.jpg)
Me it does not even offer me updates ..... Normal?

**Comment by VikeDragon** [comment]
> ![Screenshot_20220412-213856_Home Assistant](https://user-images.githubusercontent.com/40418405/163040631-38f0dc09-2cb2-42a7-be3c-68fb9a8d372e.jpg) Me it does not even offer me updates ..... Normal?

I wrote not about updating the Home Assistant functionality, but about updating data with plugins and components due to the inability to get an Internet connection from Supervisor. Since the Supervisor itself rewrites connections without taking into account routing rules and other parameters, reconfiguring the NetworkManager automatically. That is, the user configuration settings of NetworkManager are replaced by standard and simplified auto-settings that do not provide routing settings, metrics, etc.

**Comment by mdegat01** [comment]
Duplicate of #3566
